### PR TITLE
EE-778: amend heading and hyperlink text

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

File: app/views/cannotClaimRelief.scala.html [hunk 1]
```diff
@@ -28,13 +28,10 @@
...
 
-    <p>@messages(s"cannotClaimRelief.$claimant.guidance")</p>
+    <p>@messages(s"cannotClaimRelief.$claimant.guidance")
 
-    <p>
-        <a href="https://www.gov.uk/tax-relief-for-employees"
-           data-journey-click="link - click:@messages(s"cannotClaimRelief.$claimant.heading"):@messages("cannotClaimRelief.link.label")">
-            @messages("cannotClaimRelief.link.label")
-        </a>
+        <a href="https://www.gov.uk/guidance/job-expenses-for-uniforms-work-clothing-and-tools#if-your-employer-pays-towards-the-costs-of-your-expenses"
```
Comment (Pascoe121) [inline app/views/cannotClaimRelief.scala.html]: this url should be in configuration

Comment (kenneth-rayner) [reply app/views/cannotClaimRelief.scala.html]: done

File: conf/messages.en [hunk 2]
```diff
@@ -92,13 +92,13 @@ claimOnline.bullet1 = your National Insurance number
...
-cannotClaimRelief.someoneElse.title = They cannot claim tax relief for these expenses
-cannotClaimRelief.someoneElse.heading = They cannot claim tax relief for these expenses
-cannotClaimRelief.someoneElse.guidance = This is because their employer has already paid back all of their expenses.
-cannotClaimRelief.link.label = Read claim tax relief for your job expenses
+cannotClaimRelief.you.title = You cannot claim these expenses
+cannotClaimRelief.you.heading = You cannot claim these expenses
+cannotClaimRelief.you.guidance = You cannot claim these expenses because they have
+cannotClaimRelief.someoneElse.title = They cannot claim for these expenses
+cannotClaimRelief.someoneElse.heading = They cannot claim for these expenses
+cannotClaimRelief.someoneElse.guidance = You cannot claim these expenses because they have
```
Comment (Pascoe121) [inline conf/messages.en]: They cannot claim

File: conf/messages.en [hunk 3]
```diff
@@ -92,13 +92,13 @@ claimOnline.bullet1 = your National Insurance number
...
+cannotClaimRelief.link.label = all been paid back by your employer
```
Comment (Pascoe121) [inline conf/messages.en]: by their employer.

Comment (Pascoe121) [reply conf/messages.en]: you should make this link into a .you & .someoneelse link. so the content can be related to you or someone else.

File: conf/messages.en [hunk 4]
```diff
@@ -92,13 +92,14 @@ claimOnline.bullet1 = your National Insurance number
...
-cannotClaimRelief.someoneElse.title = They cannot claim tax relief for these expenses
-cannotClaimRelief.someoneElse.heading = They cannot claim tax relief for these expenses
-cannotClaimRelief.someoneElse.guidance = This is because their employer has already paid back all of their expenses.
-cannotClaimRelief.link.label = Read claim tax relief for your job expenses
+cannotClaimRelief.you.title = You cannot claim these expenses
+cannotClaimRelief.you.heading = You cannot claim these expenses
+cannotClaimRelief.you.guidance = You cannot claim these expenses because they have
+cannotClaimRelief.someoneElse.title = They cannot claim for these expenses
+cannotClaimRelief.someoneElse.heading = They cannot claim for these expenses
+cannotClaimRelief.someoneElse.guidance = They cannot claim these expenses because they have
```
Comment (Pascoe121) [inline conf/messages.en]: for these

File: conf/messages.en (hunk 3 above)
Comment (kenneth-rayner) [reply conf/messages.en]: Done

File: conf/messages.en (hunk 2 above)
Comment (kenneth-rayner) [reply conf/messages.en]: Done

File: conf/messages.en (hunk 3 above)
Comment (kenneth-rayner) [reply conf/messages.en]: Done

File: conf/messages.en (hunk 4 above)
Comment (kenneth-rayner) [reply conf/messages.en]: Done

**Pull request merged**